src/main/java/duke/command/AddTaskCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.command;
+
+import duke.Parser;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+public class AddTaskCommand extends Command {
```

Comment:
maybe it would be better if the class was named TaskAdder? AddTaskCommand sounds like maybe what a method would be called